frontend/src/employee-frontend/components/finance-basics/PlacementTypeItem.tsx

```diff
@@ -44,6 +47,7 @@ export default React.memo(function PlacementTypeItem({
           <ServiceNeedItem
             key={serviceNeed.id}
             serviceNeed={serviceNeed.nameFi + ' (oletus)'}
+            voucherValuesList={voucherValuesMap[serviceNeed.id]}
```

Comment:
If there is no voucher value for this service need yet, then I suppose this might give undefined instead of empty list? This is one of the rare cases where type system can't be trusted with its current config. Maybe `voucherValuesMap[serviceNeed.id] ?? []` could work.

Reply:
Good catch, that would seem to be an appropriate fix